pkgs/development/python-modules/mxnet/default.nix

```
@@ -9,7 +9,7 @@
 }:
 
 buildPythonPackage {
-  inherit (pkgs.mxnet) name version src meta;
+  inherit (pkgs.mxnet) name pname version src meta;
```

```suggestion
  inherit (pkgs.mxnet) pname version src meta;
```

otherwise we overwrite name possible with unexpected results if we have change the version to something else.